app/_src/guides/customizing-transparent-proxy-configuration-in-kubernetes-mode.md

```diff
@@ -0,0 +1,434 @@
+      ebpf:
+        enabled: false # KUMA_RUNTIME_KUBERNETES_INJECTOR_EBPF_ENABLED
+        instanceIPEnvVarName: INSTANCE_IP # KUMA_RUNTIME_KUBERNETES_INJECTOR_EBPF_INSTANCE_IP_ENV_VAR_NAME
+        bpffsPath: /sys/fs/bpf # KUMA_RUNTIME_KUBERNETES_INJECTOR_EBPF_BPFFS_PATH
+        cgroupPath: /sys/fs/cgroup # KUMA_RUNTIME_KUBERNETES_INJECTOR_EBPF_CGROUP_PATH
+        tcAttachIface: "" # KUMA_RUNTIME_KUBERNETES_INJECTOR_EBPF_TC_ATTACH_IFACE
+        programsSourcePath: /tmp/kuma-ebpf # KUMA_RUNTIME_KUBERNETES_INJECTOR_EBPF_PROGRAMS_SOURCE_PATH
+```
+
+### Settings Restricted to Control Plane Runtime Configuration
+
+Some transparent proxy settings **can only be changed through the control plane’s runtime configuration** because they are shared with other {{ Kuma }} components. These settings handle essential tasks like creating [`Dataplane` resources]({{ docs }}/production/dp-config/dpp/) for workloads and setting up the [`kuma-dp` sidecar container]({{ docs }}/production/dp-config/dpp-on-kubernetes/) alongside them. For example, the DNS port used to redirect traffic is shared between the transparent proxy and the `kuma-dp run` command in the sidecar container. Keeping these settings consistent across workloads prevents misconfigurations. These settings are:
+
+- `runtime.kubernetes.injector.sidecarContainer.redirectPortInbound`
+- `runtime.kubernetes.injector.sidecarContainer.redirectPortOutbound`
+- `runtime.kubernetes.injector.sidecarContainer.uid`
+- `runtime.kubernetes.injector.builtinDNS.enabled`
+- `runtime.kubernetes.injector.builtinDNS.port`
+
+{% if_version gte:2.9.x %}
+{% capture behavior-difference-danger %}
+{% danger %}
+⚠️ This behavior differs from that of other settings. Ensure you're aware of this distinction and **never** attempt to modify these settings using the mentioned annotations.
+{% enddanger %}
+{% endcapture %}
+
+If you try to change these settings with methods described later in this guide, but with values different from the runtime configuration, the effects will vary:
+
+- **`runtime.kubernetes.injector.sidecarContainer.redirectPortInbound` and `runtime.kubernetes.injector.sidecarContainer.redirectPortOutbound`**
+
+  Attempts to change these settings using `kuma.io/transparent-proxying-inbound-port` and `kuma.io/transparent-proxying-outbound-port` annotations, or by updating `redirect.inbound.port` and `redirect.outbound.port` in the ConfigMap, will trigger warnings in the control plane logs, but the values will be ignored.
+
+- **`runtime.kubernetes.injector.sidecarContainer.uid`**
+
+  Attempts to modify this setting using the `kuma.io/sidecar-uid` annotation or `kumaDPUser` in the ConfigMap will result in:
+
+  - A warning in control plane logs if the [{{ configuration-in-configmap }}](#{{ tproxy.ids.guides.customize-config.k8s.methods.configmap }}) feature is **enabled**, and the values will be ignored.
+  - Silent ignoring of the change without warnings if the feature is **disabled**.
```

Comment:
Nitpick (but maybe that's why vale is complaining)

```suggestion
  - Silently ignoring the change without warnings if the feature is **disabled**.
```

Reply:
addressed